promdens.py

```diff
@@ -566,44 +626,8 @@ def windowing(self):
 fwhm *= fstoau
 
 # checking input
-if method not in ['pda', 'pdaw']:
-    print(f"\nERROR: '{method}' is not available method!")
-    exit(1)
-
-if energy_units not in ['a.u.', 'eV', 'nm', 'cm-1']:
-    print(f"\nERROR: '{energy_units}' is not available unit for energy!")
-    exit(1)
-
-if tdm_units not in ['a.u.', 'debye']:
-    print(f"\nERROR: '{tdm_units}' is not available unit for transition dipole moment!")
-    exit(1)
-
-if ftype not in ['file']:
-    print(f"\nERROR: '{ftype}' is not available file type!")
-    exit(1)
-
-if envelope_type not in ['gauss', 'lorentz', 'sech', 'sin', 'sin2']:
-    print(f"\nERROR: '{envelope_type}' is not available envelope type!")
-    exit(1)
-
-if neg_handling not in ['error', 'ignore', 'abs']:
-    print(f"\nERROR: '{neg_handling}' is not an available option for handling negative probabilities!")
-    exit(1)
-
-if not exists(fname):
-    print(f"\nERROR: file '{fname}' not found!")
-    exit(1)
-
-if nsamples < 0:
-    print(f"\nERROR: nsamples is smaller than 0 ({nsamples})!")
-    exit(1)
-
-if new_nsamples <= 0:
-    print(f"\nERROR: npsamples is smaller than 0 ({new_nsamples})!")
-    exit(1)
-
-if nstates <= 0:
-    print(f"\nERROR: invalid number of excited states (nstates={nstates})!")
+if not Path(fname).is_file():
```

Comment:
Unlike before, this now errors out if a user provides a directory instead of a file.